core/proc.rbs

```diff
@@ -288,8 +288,184 @@
 #
 # Numbered parameters were introduced in Ruby 2.7.
 #
-class Proc < Object
-  def clone: () -> self
+class Proc
+  interface _Callable
+    def call: (*untyped, **untyped) ?{ (*untyped, **untyped) -> untyped } -> untyped
+  end
```

Comment:
```suggestion
  interface _Call
    def call: (?) -> untyped
  end
```

Reply:
Oh my, is this a new development I don't know about?